spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -143,6 +148,27 @@ public DeleteOrphanFilesSparkAction executeDeleteWith(ExecutorService executorSe
     return this;
   }
 
+  @Override
+  public DeleteOrphanFiles newPrefixMismatchMode(PrefixMismatchMode mismatchMode) {
```

Comment:
Earlier I meant using `new***` pattern for args, not methods. 

```
public DeleteOrphanFilesSparkAction prefixMismatchMode(PrefixMismatchMode newPrefixMismatchMode) {
  ...
}
```

Reply:
Got it, fixed the method and arg names